Gemfile

```diff
@@ -24,11 +24,13 @@ gem 'rails_autolink'
 # Please don't update hoptoad_notifier to airbrake.
 # It's for internal use only, and we monkeypatch certain methods
 gem 'hoptoad_notifier', "~> 2.4"
-gem 'draper', :require => false
+gem 'draper'
 
 gem 'errbit_plugin', github: 'errbit/errbit_plugin'
 gem 'errbit_github_plugin', github: 'errbit/errbit_github_plugin'
 
+gem 'dotenv-deployment'
```

Comment:
dotenv-deployment is required to use .env in production.

Capistrano 3 has a built-in feature to link files (`linked_files`) so deploy.rb does not need to include dotenv-deployment capistrano task - but still dotenv-deployment should be included in Gemfile.